ckan/controllers/user.py

```diff
@@ -156,6 +158,28 @@ def me(self, locale=None):
         user_ref = c.userobj.get_reference_preferred_for_uri()
         h.redirect_to(locale=locale, controller='user', action='dashboard')
 
+    def check_permissions(self):
+        organisations = []
+
```

Comment:
if the user isn't logged in we should just 403 this endpoint

Reply:
Fixed.